protobuf/pubspec.yaml

```diff
@@ -16,3 +16,4 @@ dev_dependencies:
   test: '>=1.2.0'
   benchmark_harness: any
   js: any
+  pedantic: ^1.8.0
```

Comment:
add trailing newline

Reply:
Also configured my editor to do this. `"files.insertFinalNewline": true,` in vscode :)